src/index.js

```diff
@@ -1,9 +1,10 @@
-module.exports.say = function say(name) 
+module.exports.say = function say(name = "world") 
 {
-  var innerName = "world";
+  let innerName = "world";
   if (typeof name !== "undefined") 
```

Comment:
🙈 empty line before

I forgot it before the logics block we like an empty line to facilitate the code reading